packages/retail-ui-extensions/src/extension-api/toast-api/toast-api.ts

```diff
@@ -0,0 +1,15 @@
+export interface ToastApiContent {
+  /**
+   * Show a toast.
+   * @param content The text content to display.
+   * @param duration The duration of the toast in ms. Defaults to 5000.
+   */
+  show: (content: string, duration?: number) => void;
```

Comment:
Using an optional positional argument for `duration` will make it very difficult to add future arguments to `toast.show()`. If this is switched to using named parameters / a description object, that will allow for future parameters without changing the method's arity.

```suggestion
  show: ({ content: string, duration?: number }) => void;
```

Reply:
actually - @olavoasantos's proposed API is preferable over this one.